api/TicketMasterAPI.js

```diff
@@ -0,0 +1,412 @@
+
+const fs = require("fs")
+const bodyParser = require("body-parser")
+const cors = require('cors');
+const express = require('express')
+const app = express()
+const port = 8080
+
+app.use(bodyParser.json())
```

Comment:
It's *usually* best practice to also include bodyParser.urlencoded({extended:true}) here as well as a matter of course - this gives us a little more versatility regarding passing information to the server from the client via url encoding.  Since we control the server and the client in this case, this could give us more flexibility as to how we interact with the API from the client as well.  [Check it out on the Express Docs for more](http://expressjs.com/en/api.html#express.urlencoded)